### DISCUSSION TITLE
"Connect to this site" popup not showing in any wallet

### DISCUSSION THREAD

**Discussion by dimitrismakrisgr** [WalletConnect/walletconnect-monorepo#2419] [Developer Support]
Hey there!

Apologies for the probably noob question, but unfortunately don't have enough experience to understand what is wrong in my setup.

I cloned the Github Repo, created a new project to WalletConnect, and I run the sample.dapps app on a Android device with the project ID I created. I have installed both MetaMask and Rainbow, but whagtever wallet I select, I don't see the popup to connect to this site.

For dapp url in AppMetaData I have set **localhost**.

Could you please provide some feedback what I am doing wrong? I saw that MetaMask doesn't support yet WalletConnect2, is this the issue with RainBow as well?

Is there any other way to Connect to MetaMask like the native OpenSea Android app for example does?

Tyvm in advance!

**Comment by cgero-eth** [comment]
Have you tried using a SSL proxy server? If I remember correctly, TrustWallet for instance does not show connection request if it comes from a non-SSL dApp.  
If you're running the sample dApp using port 3000, you can start a SSL proxy using `local-ssl-proxy`:
```
npx local-ssl-proxy --source 3001 --target 3000
```

**Comment by finessevanes** [comment]
@dimitrismakrisgr are you able to share an error code? (if you're getting one)
